srcpkgs/obs/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'obs'
 pkgname=obs
-version=22.0.3
-revision=3
-_commit_hash_obs_browser="a321928756ae8ed4315d6d7b35a113e76493297c"
-_commit_hash_obs_vst="b0f046f0ac0c8181d018f7148c7617b577a1e085"
+version=23.0.1
+revision=1
+_commit_hash_obs_browser="693498b518cf893876f454ef581e13106e8370ec"
+_commit_hash_obs_vst="fde438f7a8afdf41a7ab386a0aa1a61175fc8427"
 _commit_hash_ftl_sdk="98289f7c2ed754c647d451899b3356850c0c1fa3"
 wrksrc="obs-studio-${version}"
 archs="i686* x86_64*"
 build_style=cmake
 configure_args="-DOBS_VERSION_OVERRIDE=${version}"
-hostmakedepends="pkg-config swig"
+hostmakedepends="pkgconf swig"
```

Comment:
please use pkg-config, pkgconf is problematic

Reply:
Reverted to pkg-config.